demisto_sdk/commands/integration_diff/integration_diff_detector.py

```diff
@@ -243,22 +308,151 @@ def check_if_element_exist(element_to_check, list_of_elements, field_to_check) -
 
         return {}
 
-    def print_missing_items(self) -> bool:
+    def print_missing_items(self):
         """
         Prints the missing elements report.
+        """
+
+        for missing_type in self.missing_items_report:
+            click.secho(f"\nMissing {missing_type}:\n", fg='bright_red')
+
+            for item in self.missing_items_report[missing_type]:
+                click.secho(item['message'] + "\n", fg='bright_red')
+
+    def print_items_in_docs_format(self):
+        """
+        Prints the version differences report in docs format so the user will can copy it to the README file.
+        """
+
+        old_yaml_data = get_yaml(self.old)
+        new_yaml_data = get_yaml(self.new)
```

Comment:
you can save the 2 file data when initing that way there is no need to reload them again.

Reply:
done.